### PR TITLE
Change field Fiscal Position to be required.

### PR DESCRIPTION
**Pull request by mbcosta**
Because there is no reason to create Fiscal Rule with Fiscal Position null.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis status

**Comment by mbcosta** [comment]
Thanks @pedrobaeza  for review, I fix the PEP8 errors, change the XML file to use 4 spaces and the License Headers to new style.

File: account_fiscal_position_rule/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================================
+Account Fiscal Position Rule
+==========================================
```
Comment (pedrobaeza) [inline account_fiscal_position_rule/README.rst]: Same length of the title

File: account_fiscal_position_rule/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+This module was written to extend the functionality of ... to support ...
```
Comment (pedrobaeza) [inline account_fiscal_position_rule/README.rst]: Complete this

File: account_fiscal_position_rule/README.rst [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+and allow you to ...
+
+Installation
+============
+
+To install this module, you need to:
```
Comment (pedrobaeza) [inline account_fiscal_position_rule/README.rst]: And also the rest of the sections

File: account_fiscal_position_rule/README.rst [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+Usage
+=====
+
+To use this module, you need to:
+
+* go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
```
Comment (pedrobaeza) [inline account_fiscal_position_rule/README.rst]: Put the repo_id and remove the comment

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rvalyi** [comment]
:+1:

**Comment by renatonlima** [comment]
:+1:

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**